src/view/treeNodes/pullRequestNode.ts

```diff
@@ -280,6 +280,7 @@ export class PRNode extends TreeNode implements CommentHandler, vscode.Commentin
 			const parentFileName = change.status === GitChangeType.RENAME ? change.previousFileName! : change.fileName;
 			if (change instanceof SlimFileChange) {
 				return new RemoteFileChangeNode(
```

Comment:
@RMacfarlane from what I understand a `SlimFileChange` resulting in a `RemoteFileChangeNode` means that the file is not present locally and opening a diff won't do anything. So the command that was set in `RemoteFileChangeNode` is wrong.

I tested it out, and the current behavior when I try to open a diff of a jpg file that doesn't exist locally I just get a slient failure.

With my change, there will be an error. 

What should the behavior here be? Show a notification that says the we can't open this file?

Reply:
Yeah, I think showing a notification makes sense - we could have a button to open the file on GitHub instead